### DISCUSSION TITLE
unable to use Github Stuent pack

### DISCUSSION THREAD

**Discussion by ShallowU** [community/community#56460] [GitHub Education]
### Hi everyone,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 26, 2023 for @ShallowU.You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

Once you have resolved the issues with your sumbission, visit us again and reapply.Have questions? Join the GitHub Education Community for answers.💖 The GitHub Education Team

**Comment by samus-aran** [comment]
Hi @ShallowU 👋 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Let us know how you get on 👍
